docs/vsphere_ci/scripts/autounattend.xml

```diff
@@ -5,7 +5,7 @@
             <SetupUILanguage>
                 <UILanguage>en-US</UILanguage>
             </SetupUILanguage>
-            <InputLocale>es-ES</InputLocale>
+            <InputLocale>en-US</InputLocale>
```

Comment:
Please change the autounattend.xml file in one commit. You are forcing the reviewers to have to re-review the same file multiple times.